src/frontend/hooks/useAllObservations.ts

```diff
@@ -14,16 +15,16 @@ export const useAllObservations = (): [
   () => void
 ] => {
   const [state, dispatch] = useContext(ObservationsContext);
+  const { killState } = useContext(SecurityContext);
 
   // We store observations in state as a Map, but the components expect an array
-  const observationsArray = useMemo(
-    () =>
-      Array.from(state.observations.values()).sort((a, b) =>
-        // TODO: move sorting into component
-        a.created_at < b.created_at ? 1 : -1
-      ),
-    [state.observations]
-  );
+  const observationsArray = useMemo(() => {
+    if (killState) return [];
```

Comment:
mentioned in an earlier comment, but not entirely sure what the difference between the two state variables you defined are. based on my limited knowledge, would this be `killStateActive` instead?